system_metrics_collector/test/topic_statistics_collector/test_received_message_age.cpp

```diff
@@ -177,3 +178,14 @@ TEST(ReceivedMessageAgeTest, TestAgeMeasurement) {
   EXPECT_EQ(kExpectedMaxMilliseconds, stats.max);
   EXPECT_DOUBLE_EQ(kExpectedStandardDeviation, stats.standard_deviation);
 }
+
+TEST(ReceivedMessageAgeTest, TestGetStatNameAndUnit) {
+  TestReceivedMessageAgeCollector test_collector{};
+
+  EXPECT_EQ(
+    test_collector.GetStatisticName(),
+    topic_statistics_collector::topic_statistics_constants::kMsgAgeStatName);
+  EXPECT_EQ(
+    test_collector.GetStatisticUnit(),
+    topic_statistics_collector::topic_statistics_constants::kMillisecondUnitName);
+}
```

Comment:
This is the definition of a change detector test https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html

Let's test the string is not empty instead, or any other useful, universally true about those metrics names and units.

Reply:
Good to know! Thanks for the link :)